plugins/mcreator-localization/help/default/armor/piglin_neutral.md

```diff
@@ -0,0 +1 @@
+This condition controls whether the armor piece makes piglins around neutrals, like gold ones.
```

Comment:
I'd change the text to `This condition controls whether the armor piece makes nearby piglins neutral, like gold armor.`
Other than that LGTM